rocAL/rocAL/source/pipeline/node.cpp

```diff
@@ -23,11 +23,12 @@ THE SOFTWARE.
 #include "node.h"
 Node::~Node()
 {
-    if(!_node)
-        vxReleaseNode(&_node);
+    if (_src_tensor_roi) vxReleaseTensor(&_src_tensor_roi);
```

Comment:
I would suggest roi_tensor as part of Tensor object since it can be tied to that.  For e.g, if a node has >1 inputs which one the _src_tensor_roi correspond to?

Reply:
Done, Added the changes to have the vx_tensor for the ROI as part of tensor object
Removed the _src_tensor_roi and _dst_tensor_roi as discussed